src/components/util/VoiceMemo.js

```diff
@@ -0,0 +1,348 @@
+// // -----------------------------------------------------------------------------------------
+// // ------------------------------------ Library Imports ------------------------------------
+// // -----------------------------------------------------------------------------------------
+
```

Comment:
what is the plan with this file?

Reply:
last time you said you will continue work on this